docs/Administration.rst

```diff
@@ -3061,6 +3064,136 @@ this entry is updated:
+
+-  click on the edit icon 
+-  browse for an image in your machine
+-  click on save icon to load the image in its location
+
+|AC_image5|
+
+If you need to edit the image again, you will find the previous image name instead of the browse button with a delete
+button allowing to restore the default icon.
+
+|AC_image6|
+
+.. _ApplicationsTabAppCenter:
+
+Applications tab
+~~~~~~~~~~~~~~~~~
+
+|AC_image11|
+
+Applications tab contains:
+-  a table with the whole applications list with delete and edit actions for each line (i.e. each application)
+-  ``Add an application`` button
+
+In this table, the three radio button properties: ``Mandatory``, ``Active`` and ``Mobile``,  
+are editable through the table. You just need to switch the radio button.
+
+.. _EditApp:
+
+Edit action
+------------
+
+When you click on the edit action, a drawer opens containing the already seized properties.
+
+|AC_image12|
+
+
+.. note:: For system applications, i.e. those installed in the platform, name, URL and image fields
+          are not editable. They appear grey colored in the edit mode.
+          
+          |AC_image13|
+          
+.. _DeleteApp:
+
+Delete action
+---------------
+
+When you click on the delete action, a popup appears to confirm the application's deletion.
+
+|AC_image14|
+
+Clicking on ``Delete`` button confirms the deletions, clicking on ``Cancel`` cancels teh deletion and nothing happens.
+
+.. _AddApp:
+
+Add an application
+~~~~~~~~~~~~~~~~~~~~
+
+In the :ref:`Applications tab <ApplicationsTabAppCenter>`, you have a button "Add new application" |AC_image7|
+allowing you to open the "Add application" drawer with these fields:
+
+|AC_image9|
+
+-  ``Title``: a mandatory field allowing to define the application's name
+-  ``URL``: a mandatory field allowing to set the access URL of the application
+-  ``Image``: not mandatory, allows you to define an icon for the application, you can browse for it through your files, it
+   should have a maximum size of 100 ko, otherwise, it will not be accepted.
+-  ``Description``: not mandatory, allows you to set a description for your application to explain more its usage. 
+   This field can contain a maximum of 1000 characters.
+-  ``Mandatory``: Allows you to define if the :ref:`application is mandatory <mandatoryApps>` for end users or not, 
+   by default set to NO   
+-  ``Active``: Allows you to define if the application should be enabled or not i.e. displayed to its 
+   audience (defined in the ``Permissions`` field), by default set to "YES" i.e. active
+-  ``Mobile``: Allows you to enable or disbale the application's display in mobile devices, i.e. if you judge it a not
+   mobile friendly app, you can set this option to false and that way ot will not be dispaled in mobile devices. 
+   By default set to true.
+-  ``Permissions``: to define the application's audience, By default set to *:/platform users i.e. if you don't define
```

Comment:
*:/platform users generates a link on the * character.
Can we remove it ?

Reply:
ok